### PR TITLE
Official lang improvments

### PR DESCRIPTION
**Pull request by TrevorBivi**
- Adds a new boolean unknownExpiredDate to langProfItems to signify an unknown expired date instead of a special case for the Date column
 - Adds N/A grade option (and disables date selection for N/A grades)

### CODE REVIEW

File: services/backend/src/database/migrations/20200810152146-optional-interested-in-remote/schema.prisma [hunk 1]
```diff
@@ -485,6 +486,7 @@ model User {
   telephone            String?
   cellphone            String?
   manager              String?
+  description          String?
```
Comment (BenJeau) [inline services/backend/src/database/migrations/20200810152146-optional-interested-in-remote/schema.prisma:489]: Why is this file modified, this seem to be an old migration?

**Review by BenJeau — COMMENTED** [review]
Looks good, one small comment - testing it locally right now

**Comment by BenJeau** [comment]
Works great locally! Two small comments:

1. When something is N/A, could we ommit the (Unexpired)?
image

2. Should the default values be N/A?
image

File: services/backend/src/database/migrations/20200810152146-optional-interested-in-remote/schema.prisma (hunk 1 above)
Comment (TrevorBivi) [reply services/backend/src/database/migrations/20200810152146-optional-interested-in-remote/schema.prisma:489]: I think backend changes got merged incorrectly at some point. This schema was missing description even though it was added after the description card change. This was causing unexpected steps to be included in the migration

Comment (BenJeau) [reply services/backend/src/database/migrations/20200810152146-optional-interested-in-remote/schema.prisma:489]: Ok sounds good, just hope it won't frig up the migrations on the server :crossed_fingers:

**Comment by TrevorBivi** [comment]
> Works great locally! Two small comments:
> 
> 1. When something is N/A, could we ommit the (Unexpired)?
>    image
Good catch, fixed

> 2. Should the default values be N/A?
>    image

I don't know if it's a good idea to make the default values N/A since they would only see those form items if they checked the box saying they have have been graded. I don't think it's too common for people to be graded on some areas but not others.

**Review by BenJeau — APPROVED** [review]
LGTM
